third_party/proto/panacea/datadeal/v2/consent.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package panacea.datadeal.v2;
+
+option go_package = "github.com/medibloc/panacea-core/v2/x/datadeal/types";
```

Comment:
I changed go_package name `github.com/medibloc/panacea-core/x/datadeal/types` to `github.com/medibloc/panacea-core/v2/x/datadeal/types` (Added `v2`).
Because `panacea-core` package need to `v2` name.

Reply:
Ah... That's not good to modify imported proto files.. Let's me check if there is any other way.

Reply:
https://github.com/medibloc/panacea-core/pull/627 I made this PR in panacea-core. Let's see if this works. I adopted this practice from Osmosis.